brainpy/_src/math/ndarray.py

```diff
@@ -1033,57 +1038,43 @@ def addr(self,
       vec2 = Array(vec2)
     if not isinstance(out, Array):
       out = Array(out)
-    return _return(brainpy.math.outer(vec1, vec2, out=out))
+    return _return(jnp.outer(vec1, vec2, out=out))
```

Comment:
``jnp.outer`` does not support ``out`` argument. 

For this line, we can change it to 

```
return brainpy.math.outer(vec1, vec2, out=out)
```